CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## v2.3.0 / 2021-12-09
+
+[FEATURE]     Add a `--namespace-denylist` command line flag to exclude metrics from certain namespaces #1596 #1595 @mallow111
```

Comment:
The formatting is broken, could we perhaps add bullets points in a similar way as what we did for the previous versions?

Reply:
Sure, I completely missed the bulleted points. Added